crud/src/main/java/com/redhat/lightblue/rest/crud/AbstractCrudResource.java

```diff
@@ -78,6 +90,108 @@
         java.security.Security.setProperty("networkaddress.cache.ttl", "30");
     }
 
+    @GET
+    @Path("/search/{searchName}/{entity}")
+    public Response runSavedSearch(@PathParam("searchName") String searchName,
```

Comment:
Are searches specific to an entity?  Or could getById apply to multiple entities?